test/cppsuite/src/common/op_tracker.cpp

```diff
@@ -0,0 +1,65 @@
+/*-
```

Comment:
@tammybailey, do you agree with @ajmorton's suggestion?

Reply:
I did not put them in the `component` folder as they don't inherit from the `component` class.
Furthermore, the `perf_plotter` is a singleton like the `random_generator` which is in the `common` dir.
Regarding the `op_tracker` I am not sure. It is a class used exclusively in `bounded_cursor_perf.cpp` for now. It's more of a utility class that can help one to track the time an operation takes. Since we don't have a `utility` folder, `common` seemed the best to me.